.gitignore

```diff
@@ -1,2 +1,3 @@
 *.un~
-node_modules/
\ No newline at end of file
+node_modules/
+.vscode/
```

Comment:
I would like to keep this clean for the project. Let's remove this and just keep it in your system-level `.gitignore`.